### PR TITLE
Update redirects in Readme

### PR DESCRIPTION
**Pull request by ReadmeCritic**
Created with https://github.com/dkhamsing/frankenstein

### CODE REVIEW

**Comment by akullpp** [comment]
Really nice tool, thanks. I will continue to use it.

File: README.md [hunk 1]
```diff
@@ -252,7 +252,7 @@ A curated list of awesome Java frameworks, libraries and software.
 *Tools which handle the distribution of applications in native formats.*
 
 * [Bintray](https://bintray.com/) - Version control for binaries which handles the publishing. Can also be used with Maven or Gradle and has a free plan for open-source software or several business plans.
-* [Capsule](http://capsule.io) - Simple and powerful packaging and deployment. A fat JAR on steroids or a "Docker for Java" that supports JVM-optimized containers.
+* [Capsule](http://www.capsule.io/) - Simple and powerful packaging and deployment. A fat JAR on steroids or a "Docker for Java" that supports JVM-optimized containers.
```
Comment (akullpp) [inline README.md]: Why does it add the `www` here but removes it from `gradle.org`?

Comment (ReadmeCritic) [inline README.md]: It follows the http redirect, so what happened is
- http://www.gradle.org/ redirects to http://gradle.org/ but
- http://capsule.io redirects to http://www.capsule.io/

It is based on what the company wants its url to be :smile: does it make sense?

File: README.md [hunk 2]
```diff
@@ -557,7 +557,7 @@ A curated list of awesome Java frameworks, libraries and software.
 *Tools which substitute expressions in a template.*
 
 * [Apache Velocity](http://velocity.apache.org/) - Templates for HTML pages, emails or source code generation in general.
-* [FreeMarker](http://freemarker.org/) - General templating engine without any heavyweight or opinionated dependencies.
+* [FreeMarker](http://freemarker.incubator.apache.org/) - General templating engine without any heavyweight or opinionated dependencies.
```
Comment (akullpp) [inline README.md]: This seems odd, why add the `.incubator.` here?

Comment (ReadmeCritic) [inline README.md]: Again this is the http redirect

However some redirects are expected (should be left alone) so let me if you want me to revert this change or any other

File: README.md (hunk 1 above)
Comment (akullpp) [inline README.md]: Yes.
